staff/lab/timecard

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import argparse
+import collections
+import datetime
+
+import matplotlib.pyplot as plt
```

Comment:
@jvperrin should we also add `matplotlib` to `requirements-dev.txt`?

Reply:
Also a nice plus would be to make this work without requiring the `DISPLAY` variable set.

Reply:
Yeah, that might be good, although ideally I think utils would have its own virtualenv at some point and be installed like ocflib is, as a `dh_virtualenv` package. But until then, adding `matplotlib` to `requirements-dev.txt` makes sense.

Reply:
Ok, I added `matplotlib` to the `requirements-dev.txt`, and also fixed the output with no `$DISPLAY` set. Seems like a good idea for now, and then maybe use `dh_virtualenv` later.